csv_path_publisher_node:
  ros__parameters:
    path_topic: "/pp_path"
    csv_path: "/my_ws/src/trajectory_planning/path/sansibarstr_out.csv"